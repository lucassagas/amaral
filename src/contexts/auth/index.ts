export { AuthContext, AuthContextProvider, handleSignOut } from './auth'
