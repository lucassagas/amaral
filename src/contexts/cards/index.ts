export { CardContext, CardContextProvider } from './cards';

