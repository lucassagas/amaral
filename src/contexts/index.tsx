import { theme } from '@/styles/theme'
import { ChakraProvider } from '@chakra-ui/react'
import { ReactNode } from 'react'
import { QueryClientProvider } from 'react-query'
import { ReactQueryDevtools } from 'react-query/devtools'
import { queryClient } from 'src/lib/queryClient'
import { AuthContextProvider } from './auth'

type AppProviderProps = {
  children: ReactNode
}

export function AppProvider({ children }: AppProviderProps) {
  return (
    <ChakraProvider theme={theme}>
      <AuthContextProvider>
        <QueryClientProvider client={queryClient}>
          {children}
          <ReactQueryDevtools />
        </QueryClientProvider>
      </AuthContextProvider>
    </ChakraProvider>
  )
}
