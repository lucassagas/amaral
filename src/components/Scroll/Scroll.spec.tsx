import { AppProvider } from '@/contexts/index'
import { render } from '@testing-library/react'
import { Scroll } from '.'

describe('<PageWrapper />', () => {
  it('should render correctly', () => {
    const { container } = render(
      <AppProvider>
        <Scroll>
          <h1>ScrollComponent</h1>
        </Scroll>
      </AppProvider>
    )

    expect(container).toMatchSnapshot()
  })
})
