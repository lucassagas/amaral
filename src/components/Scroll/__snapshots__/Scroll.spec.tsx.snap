// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<PageWrapper /> should render correctly 1`] = `
<div>
  <div
    class="css-ztfaeq"
  >
    <h1>
      ScrollComponent
    </h1>
  </div>
</div>
`;
